src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,14 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService usersData = new UserService();
```

Comment:
Let's call just this variable as it is - `userService`
Because it's not just the data, it also has a functionality that we can and should use :)